soccer/motion/MotionControl.cpp

```diff
@@ -240,3 +240,17 @@ void MotionControl::_targetBodyVel(Point targetVel) {
     _robot->control->set_xvelocity(targetVel.x());
     _robot->control->set_yvelocity(targetVel.y());
 }
+
+Pid* MotionControl::getPid(char controller) {
+    // just in case there is confusion
+    switch (tolower(controller)) {
+        case 'a':
```

Comment:
This is super hacky (passing chars in here). I'm not sure how the python bridge would handle enums though, so I'll allow it for now. If you have some spare time could you look into that?